Cargo.toml

```diff
@@ -50,8 +50,21 @@
 rsconf = { git = "https://github.com/mqudsi/rsconf", branch = "master" }
 
 [lib]
-crate-type = ["staticlib"]
+crate-type = ["rlib"]
 path = "fish-rust/src/lib.rs"
+doctest = false
```

Comment:
every commit should pass tests, so this should be reordered or squashed (same for the other commits)